airflow/api_connexion/schemas/task_instance_schema.py

```diff
@@ -86,6 +87,38 @@ def get_attribute(self, obj, attr, default):
         return get_value(obj[0], attr, default)
 
 
+class TaskInstanceHistorySchema(SQLAlchemySchema):
+    """Task instance schema."""
+
+    class Meta:
+        """Meta."""
+
+        model = TaskInstanceHistory
+
+    task_id = auto_field()
+    dag_id = auto_field()
+    run_id = auto_field(data_key="dag_run_id")
```

Comment:
I'm ok with `run_id` but just want to confirm whether we need/want to change `dag_run_id` to `run_id`

Reply:
I kept it so it's not different from TaskInstanceSchema since both are similar

Reply:
got it. sounds reasonable 🙂